apps/passport-server/test/devconnect.spec.ts

```diff
@@ -0,0 +1,340 @@
+import { expect } from "chai";
+import "mocha";
+import { Pool } from "pg";
+import { getDevconnectPretixConfig } from "../src/apis/devconnectPretixAPI";
+import { IEmailAPI } from "../src/apis/emailAPI";
+import { stopApplication } from "../src/application";
+import { getDB } from "../src/database/postgresPool";
+import { fetchAllDevconnectPretixTickets } from "../src/database/queries/devconnect_pretix_tickets/fetchDevconnectPretixTicket";
+import { sqlQuery } from "../src/database/sqlQuery";
+import { DevconnectPretixSyncService } from "../src/services/devconnectPretixSyncService";
+import { PretixSyncStatus } from "../src/services/types";
+import { PCDPass } from "../src/types";
+import {
+  DevconnectPretixDataMocker,
+  EMAIL_1,
+  EMAIL_2,
+  EMAIL_3,
+  EMAIL_4,
+  EVENT_A_ID,
+  EVENT_B_ID,
+  EVENT_C_ID,
+  ITEM_1,
+  ITEM_2,
+} from "./pretix/devconnectPretixDataMocker";
+import {
+  getDevconnectMockPretixAPI,
+  MOCK_PRETIX_API_CONFIG,
+} from "./pretix/mockDevconnectPretixApi";
+import { waitForDevconnectPretixSyncStatus } from "./pretix/waitForDevconnectPretixSyncStatus";
+import { overrideEnvironment, pcdpassTestingEnv } from "./util/env";
+import { startTestingApp } from "./util/startTestingApplication";
+
+describe("devconnect configuration db tables", function () {
+  let application: PCDPass;
+
+  this.beforeAll(async () => {
+    await overrideEnvironment(pcdpassTestingEnv);
+    application = await startTestingApp();
+  });
+
+  this.afterAll(async () => {
+    await stopApplication(application);
+  });
+
+  step("test organizer config", async function () {
+    // Insert organizer 1
+    await sqlQuery(
+      application.context.dbPool,
+      "insert into pretix_organizers_config (organizer_url, token) values ('organizer-url-1', 'token1')"
```

Comment:
in the future, let's change these to be non-hardcoded, and reference the mocked api config. we should also have tests for what happens when the `pretix_organizers_config` is invalid (i.e. the mocked api should check if the organizer url and token are valid) - if that's not already happening.